test/plot-schema.json

```diff
@@ -49569,6 +49929,18 @@
        },
        "role": "object"
       },
+      "ticklabelshiftx": {
```

Comment:
This is `indicator` trace. Let's include this option in an existing mock to test it.